app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb

```diff
@@ -1,7 +1,7 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptSource
   FRIENDLY_NAME = "Embedded Ansible Project".freeze
 
-  validates :name,       :presence => true # TODO: unique within region?
+  validates :name,       :presence => true
```

Comment:
I still think this is important because of replication.  At the master region you will naturally get multiples with the same name, and if there is also embedded ansible at the global region (which is probable as people use it to blast out things from the global to the regionals), then there will name collisions.